src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentForm.java

```diff
@@ -205,8 +211,38 @@ public List<ExtraButton> getExtraButtons() {
             extraButtons.add(createCreateDVButton());
         }
         
+        createAppropriatePresentationModeChangeButton(isFullPageAllowed, isMultiplePagesAllowed);
+        
         return extraButtons;
     }
+    
+    private void createAppropriatePresentationModeChangeButton(boolean isFullPageAllowed, boolean isMultiplePagesAllowed) {
+        if (ObjectUtils.isNotNull(getPresentationMode()) && 
+               getPresentationMode().equalsIgnoreCase(CUPurapConstants.IWantPresentationModes.MULTIPLE_PAGE_MODE) &&
+               isFullPageAllowed && isUsersRequestingPresentationModeChange()) {
+            extraButtons.add(createShowFullPageButton());
+        } else if (ObjectUtils.isNotNull(getPresentationMode()) && 
+                getPresentationMode().equalsIgnoreCase(CUPurapConstants.IWantPresentationModes.FULL_PAGE_MODE) &&
+                isMultiplePagesAllowed && isUsersRequestingPresentationModeChange()) {
+            extraButtons.add(createShowMultiplePagesButton());
+        }
+    }
+    
+    protected ExtraButton createShowFullPageButton() {
+        ExtraButton showFullPageButton = new ExtraButton();
+        showFullPageButton.setExtraButtonProperty("methodToCall.showFullPage");
+        showFullPageButton.setExtraButtonSource("${" + KFSConstants.EXTERNALIZABLE_IMAGES_URL_KEY + "}buttonsmall_continue.gif");
+        showFullPageButton.setExtraButtonAltText("Show Full Page");
```

Comment:
This button name doesn't line up with the yellow-highlighted one that was mentioned in the new user story. Can you please update this if necessary?

The same goes for the other new button.